docker-compose.yml

```diff
@@ -429,6 +429,13 @@ services:
             - graph-node
         volumes:
             - data-graph-deploy:/firstrun
+    graph-deploy-dataunion-subgraph:
+        container_name: streamr-dev-graph-deploy-dataunion-subgraph
+        image: streamr/graph-deploy-dataunion-subgraph:dev
+        depends_on:
+            - graph-node
+        volumes:
+            - data-graph-deploy-dataunion:/firstrun
```

Comment:
Let's use the [long syntax for volumes](https://docs.docker.com/compose/compose-file/compose-file-v3/#long-syntax-3).

Including relevant attributes:
```
read_only: [boolean]
bind:
    propagation: [type]
volume:
    nocopy: [boolean]
```